docs/src/pages/guides/FHIRServerUsersGuide.md

```diff
@@ -1296,6 +1296,8 @@ psql -v ON_ERROR_STOP=1 -1 -U javabatch -f batchPersistence-postgresql.ddl -d jo
 </server>
 ```
 
+Note: If you use Apache PostgreSql database as FHIR data store or JavaBatch job repository, please enable max_prepared_transactions in postgresql.conf, otherwise the import/export JavaBatch jobs will fail.
```

Comment:
```suggestion
Note: If you use PostgreSql database as IBM FHIR Server data store or the JavaBatch job repository, please enable `max_prepared_transactions` in postgresql.conf, otherwise the import/export JavaBatch jobs fail.
```